### PR TITLE
Largest Rectangle in Histogram 

### PR DESCRIPTION
**Pull request by shubhm-cmd**
Added a question from leetcode

### CODE REVIEW

File: Task 1/CHEFEZQ/Question.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Chef published a blog post, and is now receiving many queries about it. On day i, he receives Qi queries. But Chef can answer at most k queries in a single day.
+
+Chef always answers the maximum number of questions that he can on any given day (note however that this cannot be more than k). The remaining questions (if any) will be carried over to the next day.
+
+Fortunately, after n days, the queries have stopped. Chef would like to know the first day during which he has some free time, i.e. the first day when he answered less than k questions.
+
+Input:
```
Comment (nimishbongale) [inline Task 1/CHEFEZQ/Question.md:7]: You can use the markdown heading formatting for sub headings. Something like this:

```md
#### Input
```
which would render as:

#### Input

File: Task 1/POSAND/Question.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+A permutation p1,p2...pN of {1,2,...,N} is beautiful if pi&pi+1 is greater than 0 for every 1≤i<N . You are given an integer N, and your task is toconstruct a beautiful permutation of length N or determine that it's impossible.
+
+Note that a&b denotes the bitwise AND of a and b.
+
+Input:
```
Comment (nimishbongale) [inline Task 1/POSAND/Question.md:5]: Here as well, if possible! It doesn't stop me from merging your PR, but I think its a small change which you would be able to do!

File: Task 1/RAINBOWA/Question.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Next a4 elements equal 4.
+Next a3 elements equal 3.
+Next a2 elements equal 2.
+Next a1 elements equal 1.
+ai can be any non-zero positive integer.
+There are no other elements in array.
+
+Help Chef in finding out if the given array is a Rainbow Array or not.
+
+Input
```
Comment (nimishbongale) [inline Task 1/RAINBOWA/Question.md:23]: Here also 👍

**Review by nimishbongale — COMMENTED** [review]
Good job @shubhm-cmd!

Very tiny change suggestions. Let's get this merged!

File: Task 1/RAINBOWA/Question.md (hunk 3 above)
Comment (shubhm-cmd) [reply Task 1/RAINBOWA/Question.md:23]: Oh I just got to know about this I will do these changes thankyou

**Review by nimishbongale — APPROVED** [review]
Awesome work @shubhm-cmd! 🚀 

You are now ready to contribute to cool open source projects! 😄

**Pull request merged**